### ISSUE TITLE
Quake mode window positioning under Wayland

### ISSUE THREAD

**Issue by gnunn1** [gnunn1/terminix#494]
The window positioning for quake mode under Wayland seems to be broken at the moment though I could have sworn I had it working at one point. I need to review my past commits to see where I may have broken it, in particular the fixes I did for Ubuntu and quake mode positioning.

Also, reading more about this it's unclear that Wayland actually supports global positioning which is what quake mode requires. If I can't get it working I may need to disable the feature under Wayland which would be a shame.

**Comment by gnunn1** [comment]
So apparently this worked previously when I tested it because I didn't have the width option available. I suspect that when the width of the window equals the screen width the Mutter/Wayland default positioning for the window is 0,0 which is why it appears to work. Once the width is less then 100% of the screen Mutter/Wayland positions the window itself and ignores my move request.

I've kludged fixed this for now by simply always using a 100% width when running under Wayland. I was wondering if a few folks who are running Wayland could test it and see if it positions the window at 0,0 for them as well.

I am seriously considering disabling quake mode when running under Wayland, there's just too many issues at this time and I'm not at all confident about it. The sort of kludge I'm doing here feels totally wrong.
